docs/how-to.md

```diff
@@ -0,0 +1,4 @@
+# How to use..
```

Comment:
I think this branch might need a rebase to include the updated documentation if it's already in master, or have the other PR merged in then rebased here.

I've approved the earlier PR to merge in, and just clicked to merge it in. You might need to try rebasing this branch to include the updates docs from it.

The link below shows the link to the me approving your earlier documentation work, and I mentioned you're free to merge it in, but it might not have been obvious.

https://github.com/thegreenwebfoundation/admin-portal/pull/231#pullrequestreview-942040899